src/app/cli/src/commands/system_diagnose_command.rs

```diff
@@ -0,0 +1,246 @@
+        write!(f, "container installed... ")?;
+
+        let container_installed_check_status = match self
+            .container_runtime
+            .custom_cmd("--version".to_string())
+            .output()
+            .await
+        {
+            Ok(_) => style("ok").green(),
+            Err(_) => style("failed").red(),
+        };
+
+        write!(f, "{container_installed_check_status}")?;
+        Ok(())
+    }
+
+    async fn container_pull_check(&self, f: &mut dyn std::io::Write) -> Result<(), CLIError> {
+        write!(f, "container can pull images... ")?;
+
+        let is_container_image_pulled =
+            match self.container_runtime.pull_image(DUMMY_IMAGE, None).await {
+                Ok(_) => style("ok").green(),
+                Err(_) => style("failed").red(),
+            };
+
+        write!(f, "{is_container_image_pulled}")?;
+        Ok(())
+    }
+
+    async fn container_rootless_run_check(
+        &self,
+        f: &mut dyn std::io::Write,
+    ) -> Result<(), CLIError> {
+        write!(f, "container rootless run check... ")?;
+
+        let container_rootless_check_status = match self
+            .container_runtime
+            .run_attached(DUMMY_IMAGE)
+            .init(true)
+            .spawn()
+            .unwrap()
+            .wait()
+            .await
+        {
+            Ok(_) => style("ok").green(),
+            Err(_) => style("failed").red(),
+        };
+
+        write!(f, "{container_rootless_check_status}")?;
+        Ok(())
+    }
+
+    async fn container_volume_mount_check(
+        &self,
+        f: &mut dyn std::io::Write,
+    ) -> Result<(), CLIError> {
+        write!(f, "container volume mounts work... ")?;
+        let temp_dir = tempfile::tempdir()?;
+        let cwd = Path::new(".").canonicalize()?;
+
+        let file_path = temp_dir.path().join("tmp.txt");
+        let _ = File::create(file_path.clone())?;
+
+        let container_volume_check_status = match self
+            .container_runtime
+            .run_attached(DUMMY_IMAGE)
+            .volume((cwd, temp_dir.into_path()))
+            .init(true)
+            .spawn()
+            .unwrap()
+            .wait()
+            .await
+        {
+            Ok(_) => style("ok").green(),
+            Err(_) => style("failed").red(),
+        };
+
+        fs::remove_file(file_path)?;
+
+        write!(f, "{container_volume_check_status}")?;
+        Ok(())
+    }
+
+    async fn workspace_consistency_check(
+        &self,
+        listener: Option<Arc<VerificationMultiProgress>>,
+        f: &mut dyn std::io::Write,
+    ) -> Result<(), CLIError> {
+        write!(f, "workspace consistent... ")?;
+        let datasets: Vec<_> = self.dataset_repo.get_all_datasets().try_collect().await?;
+
+        let verify_options = VerificationOptions {
+            check_integrity: true,
+            replay_transformations: false,
+        };
```

Comment:
When talking to @zaychenko-sergei we realized that currently these options will include verification of both physical and logical hashes of data.

Logical hashing is pretty slow currently and can take minutes on large data. I think we should extend `VerificationOptions` with `check_logical_hashes` flag and skip this validation during `system diagnose` command.